tailwind.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     extend: {
       colors: {
         //notice the convenient naming...
-        gray: colors.slate,
+        gray: { 50: '#F8FAFC', 100: '#F1F5F9', 600: '#344154', 800: '#0F1729' },
```

Comment:
You can use the default colors palette like we had before and adjust only the needed shades like->
`gray: { ...colors.slate, 50: '#F8FAFC', 100: '#F1F5F9', 600: '#344154', 800: '#0F1729' },`

Reply:
I thought colors.slate was already being used by default, since gray shades I haven't declared were working properly
and since it wasn't created or modified anywhere in the src 
Using a color inspector, I see that being with or without colors.slate produces a slightly different color!